types/meteor/globals/mongo.d.ts

```diff
@@ -239,3 +390,14 @@ declare module Mongo {
         transform?: Function | null;
     }
 }
+
+declare interface MongoConnection {
+    db: any;
+    client: any;
+}
```

Comment:
Why `any`? It works correctly in `types/meteor/mongo.d.ts` with proper types:
```ts
    interface MongoConnection {
        db: MongoDb;
        client: MongoClient;
    }
```

Reply:
I tried it with `/// <reference types="mongodb" />`
but it didn't work so I described it as `any;` to get further.